src/components/WebexAddCollaborators/WebexAddCollaborators.jsx

```diff
@@ -1,19 +1,85 @@
-import React from 'react';
-
-import {InputField} from '../generic';
+import React, {useState, useEffect} from 'react';
+import {InputField, OptionsList} from '../generic';
+import {useAddCollaborators} from '../hooks';
+import webexComponentClasses from '../helpers';
 
 /**
  * Webex Add Collaborators component
  *
+ * @returns {object} JSX of component
  */
 export default function WebexAddCollaborators() {
+  const [inputValue, setInputValue] = useState('');
+  const [selected, setSelected] = useState('');
+  const [cssClasses, sc] = webexComponentClasses('add-collaborators');
+  const searchList = useAddCollaborators(inputValue);
+
+  useEffect(() => {
+  }, [inputValue]);
+
+  const handleOnChange = (value) => {
+    setInputValue(value);
+  };
+
+  const handleOnSelect = (opt) => {
+    setSelected(opt.value);
+  };
+
+  const renderSuggestions = () => {
+    if (!searchList) return null;
+    const content = searchList.map((item) => {
+      const names = item.displayName.split(' ');
+      const initials =
+        names.length === 1
+          ? names[0][0]
+          : `${names[0][0]}${names[names.length - 1][0]}`;
+
+      return ({
+        value: item.emails[0],
```

Comment:
Same as above.Can we change everywhere where trying to access the item object fields.

Reply:
changes done